src/d3a/models/area/stats.py

```diff
@@ -118,11 +126,7 @@ def get_price_stats_current_market(self):
                 if self.current_market.time_slot in self.rate_stats_market else None
 
     def min_max_avg_median_rate_current_market(self):
-        out_dict = {"min_trade_rate": None,
-                    "max_trade_rate": None,
-                    "avg_trade_rate": None,
-                    "median_trade_rate": None,
-                    "total_traded_energy_kWh": None}
+        out_dict = deepcopy(default_trade_stats_dict)
```

Comment:
Performance impact is small, but please use a shallow copy here (essentially only `copy` and not `deepcopy`) because there are no references in the dict, and even a shallow copy will work fine.